apps/api/schedule.py

```diff
@@ -216,6 +216,20 @@ def post(self):
                     )
                     proposal.c3voc_url = None
 
+                if payload["voctoweb"]["thumb_path"]:
+                    path = payload["voctoweb"]["thumb_path"]
+                    if path.startswith("/static.media.ccc.de"):
+                        path = "https://static.media.ccc.de/media" + path[len("/static.media.ccc.de"):]
+                    if not path.startswith("https://"):
```

Comment:
Are other CDNs likely to be used to host these thumbnails? (just wondering about the broader URI acceptance logic in this case)

Reply:
This URL has changed in the past (some time ago it was `cdn.c3voc.de` i think). I don't think this will change in the future though, so we could have implemented stricter checking here, you're right.

Reply:
Thanks, OK!  For consistency and safety, I think it makes sense to check the complete domain in that case too.